regolith/helpers/makeappointmentshelper.py

```diff
@@ -0,0 +1,126 @@
+"""Helper for managing appointments.
+
+   - Returns members with gap in appointments
+   - Returns members supported on an outdated grant
+   - Returns members supported on a depleted grant
+   - Suggests appointments to make for these members
+   - Suggests new appointments
+"""
+import datetime as dt
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+from regolith.dates import get_due_date
+from regolith.helpers.basehelper import SoutHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+    is_fully_appointed
+)
+from regolith.dates import (
+    is_current,
+    has_finished,
+)
+
+TARGET_COLL = "people"
+HELPER_TARGET = "makeappointments"
+
+
+def subparser(subpi):
+
+    subpi.add_argument("-v", "--verbose", action="store_true", help='increase verbosity of output')
+    #subpi.add_argument("-g", "--gap", action="store_true",
+    #                   help='get group members with a gap in their appointments')
+    #subpi.add_argument("-o", "--out_of_date", action="store_true",
+    #                   help='get group members supported on an out of date grant')
+    #subpi.add_argument("-d", "--depleted", action="store_true",
+    #                   help='get group members supported on a depleted grant')
+    subpi.add_argument("-b", "--begin_date", help='begin date of interval to check appointment, to specify')
+    subpi.add_argument("-e", "--end_date", help='end date of interval to check appointments')
+    subpi.add_argument("-o", "--overspent", action="store_true", help='return grants that are overspent by more' 
```

Comment:
I think I want this info without setting any flag.  Same for underspent

Reply:
done